features/0023-did-exchange/README.md

```diff
@@ -298,13 +301,17 @@ When a `request` responds to an implicit invitation, its `~thread.pthid` MUST co
   "@type": "https://didcomm.org/didexchange/1.0/request",
   "~thread": { "pthid": "032fbd19-f6fd-48c5-9197-ba9a47040470" },
   "label": "Bob",
-  "connection": {
-    "did": "B.did@B:A",
-    "did_doc": {
-        "@context": "https://w3id.org/did/v1"
-        // DID Doc contents here.
-    }
-  }
+  "did": "B.did@B:A",
+  "did_doc~attach": {
+     "base64": "eyJ0eXAiOiJKV1Qi... (bytes omitted)",
+     "jws": {
+        "header": {
+           "kid": "did:key:z6MkmjY8GnV5i9YTDtPETC2uUAW6ejw3nk5mXF5yci5ab7th"
```

Comment:
should this also include the key fragment of the signing key?

Reply:
in did key, there is only one key. Should a fragment be needed there?

Reply:
did:key with ed25519 has 2 keys. One signing key (ed25519) and one key agreement key (x25519) that's derived from the signing key. 

Notice the fragments are different in this example: https://w3c-ccg.github.io/did-method-key/#example-2-a-did-document-derived-from-a-did-key

it's because it's the fingerprint of the publickey. 

Here's an example of the implementation where it does the deriving: https://github.com/digitalbazaar/did-method-key-js/blob/master/lib/driver.js#L104